config/navigations/services_navigation.rb

```diff
@@ -286,11 +286,11 @@
                  'Services',
                  nil,
                  html: { class: 'fancy-nav-header', 'data-icon': 'service-icon' },
-                 if: -> { services.available?(:email_service) } do |services_nav|
+                 if: -> { plugin_available?(:email_service) } do |services_nav|
       services_nav.item :email_service,
-                        'Email',
+                        'eMail',
                         -> { plugin('email_service').index_path },
-                        if: -> { services.available?(:email_service) },
+                        if: -> { plugin_available?(:email_service) },
```

Comment:
The same as before here.

Reply:
done.

Reply:
I need to validate using `plugin_available?` method now as there is no `email_service` registered now.

Reply:
this just check if there is a plugin called like this and the plugin exists already. Keep it.